src/DatePicker/DateDisplay.js

```diff
@@ -47,7 +47,7 @@ function getStyles(props, context, state) {
       marginBottom: 10,
     },
     yearTitle: {
-      cursor: (!selectedYear && !props.disableYearSelection) ? 'pointer' : 'default',
+      cursor: props.disableYearSelection ? 'not-allowed' : (!selectedYear ? 'pointer' : 'default'),
```

Comment:
> Changing the `disableYearSelection` property doesn't change the styling of the element.

good point, agree. But I think, in this particular case we **should** emphasize disabled case.
Property `disableYearSelection` can be dependent on some custom application component property.
And this is case for the emphasizing -  in come cases this field will be `true`, in some `false`. And year field in UI will be interactive or not.